config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/konveyor/oadp-operator
-  newTag: latest
+  newName: ttl.sh/oadp-operator-967a78f
```

Comment:
Oppsies.. you probably `git add .`

These were most likely coming from `make deploy-olm`.
It can be reverted by running `make bundle` again.
Alternatively we could add `make bundle` to the end of `make deploy-olm`

Reply:
ahh thanks. This file always changes even with `IMG=<> make deploy` I just usually ignore it